### DISCUSSION TITLE
Can destination directory use network drive?

### DISCUSSION THREAD

**Discussion by fantasiataiwan** [swar/Swar-Chia-Plot-Manager#217] [Q&A] [answered]
I use network drive line Z:\ form my NAS but it doesn't work!

**Comment by swar** [comment, marked as answer]
No, use the regular network path. "//NetworkPath//folder"

**Reply by DomChia** [reply]
Through the GUI its working, bun through SWAR not work... tryed many different ways how to type destination...

**Comment by harley19801** [comment]
Yes, you can map it in Windows and use as destinantion drive. Nice bonus that you can change it and map another drive to the same letter and plots in process will be rerouted to the new drive on the fly
